source/upgrade.txt

```diff
@@ -52,6 +52,27 @@ changes between the current and upgrade versions. For example, if you
 are upgrading the driver from v4.0 to v4.5, address all breaking changes from
 the version after v4.0 including any listed under v4.5.
 
+.. _java-breaking-changes-v5.0:
+
+Version 5.0 Breaking Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Several changes were made to the ``ConnectionId`` class.
+  
+  The ``ConnectionId`` constructor now accepts a value of type ``long`` as its second
+  parameter instead of type ``int``. Similarly, the constructor now accepts a value of
+  type ``Long`` as its third parameter instead of type ``Integer``. Since this change breaks
```

Comment:
[Style Guide](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#std-term-since): Don't use "since" to explain why, instead use "because"

```suggestion
  type ``Long`` as its third parameter instead of type ``Integer``. Because this change breaks
```